gui/src/installer/view.rs

```diff
@@ -414,7 +414,19 @@ pub fn import_descriptor<'a>(
                     } else {
                         None
                     })
-                    .push(col_descriptor),
+                    .push(col_descriptor)
+                    .push_maybe(if change_network {
+                        // only show message when importing a descriptor
+                        Some(Row::new().push(text(
```

Comment:
You can remove useless Row::new()

Reply:
Ah yes, thanks for spotting. Removed that now.